<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorFsxConfigWindow</class>
 <widget class="QWidget" name="CSimulatorFsxConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FSX/P3D plugin settings</string>
  </property>
  <layout class="QVBoxLayout" name="vl_FsxConfigWindow">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tw_Settings">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_SimConnectConfig">
      <attribute name="title">
       <string>SimConnect config</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_SimConnectSettings">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="swift::simplugin::fsxcommon::CSimConnectSettingsComponent" name="comp_SimConnectSettings"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_FsxSettings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Settings">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="swift::simplugin::fsxcommon::CFsxSettingsComponent" name="comp_FsxSettings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_OkCancel">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::simplugin::fsxcommon::CSimConnectSettingsComponent</class>
   <extends>QFrame</extends>
   <header>simconnectsettingscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::simplugin::fsxcommon::CFsxSettingsComponent</class>
   <extends>QFrame</extends>
   <header>fsxsettingscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
